Shaders/MXAO 4.0.2 EX.fx

```diff
@@ -24,75 +24,76 @@
 #endif
 
 #ifndef MXAO_SMOOTHNORMALS
- #define MXAO_SMOOTHNORMALS             1       //[0 or 1]      This feature makes low poly surfaces smoother, especially useful on older games.
```

Comment:
This is intentionally set to 1 in GShade, as the primary concern of the build is Final Fantasy XIV support.